src/pointermap.js

```diff
@@ -1,56 +1,47 @@
 /**
  * This module implements an map of pointer states
  */
-var USE_MAP = window.Map && window.Map.prototype.forEach;
-var POINTERS_FN = function() { return this.size; };
+var USE_MAP = window.Map && window.Map.prototype.forEach &&
+  !Object.getOwnPropertyDescriptor(Map.prototype,'size').value;
```

Comment:
That's silly, why would we care about something that was fixed almost three years ago? Remove this.